src/app/medical_record/repositories/mysql.go

```diff
@@ -84,6 +96,19 @@ func (repo *repository) SelectDataByPatientID(id string) ([]medicalrecord.Domain
 	return MapToBatchDomain(records), nil
 }
 
+// UpdateByID implements medicalrecord.Repositories
+func (repo *repository) UpdateByID(domain medicalrecord.Domain, id string) error {
+	record := MapToExistingRecord(domain)
+	query := repo.DB.Where("ID = ?", id).Omit("ID").Updates(&record)
+
+	println(id, query)

```

Comment:
forgot to remove this? hehe

Reply:
Applied https://github.com/ALTERA-CAPSTON-41/backend/pull/68/commits/a2ec4c956c0bff8fab94ec05c4806926dc37f6b3.